CONTRIBUTING.md

```diff
@@ -24,10 +24,10 @@ Clone the repo `git clone https://github.com/getsentry/sentry-unity.git` and `cd
 dotnet msbuild /t:"UnityPlayModeTest;UnityEditModeTest" /p:Configuration=Release
 ```
 
-### Smoke test by building and running a player with IL2CPP::
+### Smoke test by building and running a player with IL2CPP:
 
 ```sh
- dotnet msbuild /t:"UnityBuildPlayerIL2CPP;UnitySmokeTestPlayerIL2CPP"
+ dotnet msbuild /t:"Build;UnityBuildStandalonePlayerIL2CPP;UnitySmokeTestStandalonePlayerIL2CPP"
```

Comment:
After `dotnet clean`, we need to `dotnet build` before running these custom targets (otherwise it fails due to missing `Sentry.Unity` and removes the `.meta`, etc.

One option is to set the `DependsOn` property so that we can run only the final target but that makes it harder to run them separately. That is, unless we add proper support for incremental builds:

https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-targets?view=vs-2019#incremental-builds